documentation/book/proc-uninstalling.adoc

```diff
@@ -0,0 +1,29 @@
+// This module is included in the following assemblies:
+//
+// master.adoc
+
+[id='ininstalling-{context}']
+= Uninstalling {ProductName}
+
```

Comment:
You need the Procedure heading per the mod docs template.

Reply:
Also, the following doesn't really feel like a procedure. You want to provide step by step instructions - maybe an intro and some prereqs as well.

Reply:
I looked at AMQ online and copied their minimal approach for uninstall, but I will add an intro and look at what we might need for prereqs.

Reply:
Actually, step one is more a prereq. I will move and see if it makes it clearer.